dev/flat_clean_fusen_files.Rmd

```diff
@@ -219,18 +217,33 @@ test_that("register_all_to_config can be run twice", {
 test_that("config file is correctly built after half register", {
   out_actual <- yaml::read_yaml(file.path(dummypackage, "dev", "config_fusen.yaml"))
   # To update
-  # file.copy(file.path(dummypackage, "dev", "config_fusen.yaml"), here::here("tests/testthat/config_fusen_register.yaml"))
+  # file.copy(file.path(dummypackage, "dev", "config_fusen.yaml"), here::here("tests/testthat/config_fusen_register.yaml"), overwrite = TRUE)
   if (file.exists("config_fusen_register.yaml")) {
     out_expected <- yaml::read_yaml("config_fusen_register.yaml")
   } else {
     # during dev in root directory
     out_expected <- yaml::read_yaml(here::here("tests/testthat/config_fusen_register.yaml"))
   }
 
+  for (flat_name in c("flat_full.Rmd", "flat_new_one.Rmd")) {
+    out_actual[[flat_name]][["inflate"]][["flat_file"]] <- basename(
+      out_actual[[flat_name]][["inflate"]][["flat_file"]]
+    )
+
+    out_expected[[flat_name]][["inflate"]][["flat_file"]] <- basename(
+      out_expected[[flat_name]][["inflate"]][["flat_file"]]
+    )
+
+    out_actual[[flat_name]][["inflate"]][["pkg"]] <- "tmp_pkg"
```

Comment:
Same here, I am not sure why you need to force a specific name.  
What would you expect to store in the config file in this place exactly ?
`pkg` is supposed to be the folder of the project. I know that it can be different for different developers, but usually, it is the name of the package. I guess in the config file, you can stored the basename `basename(normalizePath(pkg))` maybe ?

Reply:
See previous comment.